.goreleaser.yml

```diff
@@ -20,19 +19,19 @@ builds:
     ldflags:
       - -s -w -X main.commit={{.Commit}} -X main.date={{ .CommitDate }} -X github.com/cosmos/cosmos-sdk/version.Name=umee -X github.com/cosmos/cosmos-sdk/version.AppName=umeed -X github.com/cosmos/cosmos-sdk/version.Version={{ .Version }} -X github.com/cosmos/cosmos-sdk/version.Commit={{ .Commit }} -X github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger -X github.com/tendermint/tendermint/version.TMCoreSemVer={{ .Env.TM_VERSION }}
     goos:
-      - darwin
       - linux
     goarch:
       - amd64
-      - arm64
+#      - arm64  # github only supports linux @ amd64
 
 archives:
-  - id: bins
-    format: binary
-    name_template: "{{ .Binary }}-v{{ .Version }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+  # - id: bins
+  #   format: binary
+  #   name_template: "{{ .Binary }}-v{{ .Version }}-{{ .Os }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
 
   - id: tarball
     format: tar.gz
+    wrap_in_directory: true
```

Comment:
Wasn't this option related to why cosmovisor was failing on `.tar.gz`? I can't remember if it was failing on `true` or `false` though.

Reply:
Don't remember as well, if we go with the route of archives releases, I recommend testing that with cosmovisor and updating the readme indication to add the env variable 
@robert-zaremba

Reply:
OK

Reply:
confirmed, should be false.